src/components/Badge.test.tsx

```diff
@@ -270,19 +279,21 @@ describe("BadgeMaybeAbsent", () => {
   });
 
   it("renders string as text", () => {
-    const { rerender } = render(<BadgeMaybeAbsent text="Power" />);
+    const { rerender } = render(<BadgeMaybeAbsent>Power</BadgeMaybeAbsent>);
     expect(screen.getByText("Power")).toBeInTheDocument();
-    rerender(<BadgeMaybeAbsent text={2030} />);
+    rerender(<BadgeMaybeAbsent>2030</BadgeMaybeAbsent>);
```

Comment:
Similar to the above: why was the numeric previously in squiggles, and this one isn't?